### PR TITLE
Incorporate Latest 1.18.2 Fabric Changes

### PR DESCRIPTION
**Pull request by peaceheis**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -13,8 +13,8 @@ plugins {
 	id "org.quiltmc.gradle.licenser" version "1.1.1"
 	id "org.ajoberstar.grgit" version "3.1.0"
 	id "com.matthewprenger.cursegradle" version "1.4.0"
+	id "me.modmuss50.remotesign" version "0.3.0" apply false
 	id "com.modrinth.minotaur" version "2.2.0"
```
Comment (EnnuiL) [inline build.gradle]: Don't shuffle the order here, it'll confuse Git diffs
```suggestion
	id "com.modrinth.minotaur" version "2.2.0"
	id "me.modmuss50.remotesign" version "0.3.0" apply false
```

**Review by LambdAurora — APPROVED** [review]
Let's get this over with.

File: gradle.properties [hunk 2]
```diff
@@ -1,9 +1,8 @@
 org.gradle.jvmargs=-Xmx2560M
```
Comment (EnnuiL) [inline gradle.properties]: Soooo, QFAPI already had it but FAPI added it recently; plus + plus != minus
```suggestion
org.gradle.jvmargs=-Xmx2560M
org.gradle.parallel=true
```

**Review by EnnuiL — CHANGES_REQUESTED** [review]
Few changes before we merge this

**Review by EnnuiL — APPROVED** [review]
Ennui appeased with success; Thanks for contributing!

**Pull request merged**